android/src/main/java/com/swmansion/rnscreens/ScreenContainer.java

```diff
@@ -151,6 +151,16 @@ protected Screen getScreenAt(int index) {
     return mScreenFragments.get(index).getScreen();
   }
 
+  @Nullable
+  public Screen getTopScreen() {
```

Comment:
We usually put `@Nullable` next to the returned type, i.e. `public @Nullable String` as "nullable" is a return type modifier rather than a method modifier